android/KMEA/app/src/main/java/com/tavultesoft/kmea/KMManager.java

```diff
@@ -102,6 +102,7 @@
   private static boolean debugMode = false;
   private static boolean shouldAllowSetKeyboard = true;
   private static boolean didCopyAssets = false;
+  private static boolean testmode = false;
```

Comment:
Instead of `testmode`, I wonder if we can set a `systemProperty` in `build.gradle` so it will apply for all the unit tests

See https://developer.android.com/studio/test#test_options

something like:
```
android {
  ...
  testOptions {
  ...
  unitTests {
    all {
      systemProperty 'somethingTestMode' 'true'
     }
    }
  }
}